wwwroot/js/adyenImplementation.js

```diff
@@ -25,52 +53,27 @@ async function initCheckout() {
             currency: "EUR",
           },
         },
-      },
-      onSubmit: (state, component) => {
-        if (state.isValid) {
-          handleSubmission(state, component, "/api/initiatePayment");
+        paypal: {
+          amount: {
+            value: 1000,
+            currency: "USD",
+          },
+          environment: "test", // Change this to "live" when you're ready to accept live PayPal payments
+          countryCode: "US", // Only needed for test. This will be automatically retrieved when you are in production.
         }
       },
-      onAdditionalDetails: (state, component) => {
-        handleSubmission(state, component, "/api/submitAdditionalDetails");
```

Comment:
Isn't this still required to [confirm additional actions](https://docs.adyen.com/online-payments/web-components/advanced-use-cases/additional-actions)?

Reply:
It was my understanding that it was required in some cases such as QR codes for Swish, but perhaps it is optional.

Reply:
oh, it is definitely still there! We have not removed any functionality, but it is implemented differently in some cases. You can see the call to `submitDetails` in adyenImplemention.js